<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>confocal</class>
 <widget class="QWidget" name="confocal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="confocal_layout" stretch="1,5">
     <item>
      <widget class="QGroupBox" name="confocal_control_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Confocal Image</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="image_button">
          <property name="text">
           <string>Image</string>
          </property>
          <property name="shortcut">
           <string>Alt+I</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop_button">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));</string>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="set_scan_button">
          <property name="text">
           <string>Set scan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="filename_label">
          <property name="text">
           <string>IMAGE NAME</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="save_layout">
          <item>
           <widget class="QPushButton" name="save_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../../artwork/qudi_icons.qrc">
              <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../../artwork/qudi_icons.qrc">
              <normaloff>:/icons/icons/delete.png</normaloff>:/icons/icons/delete.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="load_layout">
          <item>
           <widget class="QPushButton" name="load_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../../artwork/qudi_icons.qrc">
              <normaloff>:/icons/icons/load.png</normaloff>:/icons/icons/load.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="previous_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../../artwork/qudi_icons.qrc">
              <normaloff>:/icons/icons/previous.png</normaloff>:/icons/icons/previous.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="next_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../../artwork/qudi_icons.qrc">
              <normaloff>:/icons/icons/next.png</normaloff>:/icons/icons/next.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="img_layout">
       <item>
        <layout class="QVBoxLayout" name="img_fw_layout"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="img_bw_layout"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../artwork/qudi_icons.qrc"/>
 </resources>
 <connections/>
</ui>
